plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -663,16 +665,29 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
       return false
     }
 
+    private val onMapIdleListener: OnMapIdleListener = object : OnMapIdleListener {
+      override fun onMapIdle() {
+        mapListenerDelegate.removeOnMapIdleListener(this)
+        // Remove dragging annotation from drag layer
+        handler.postDelayed(
+          {
+            // Delay 1 second to avoid blink issue.
+            delegateProvider.getStyle { style ->
```

Comment:
can we get crash when remove (set to null) annotation manager before this postDelayed happen?
should we remove it from handler?

Reply:
Not see a crash when test this scenario locally. But need remove it in `onDestroy`.